src/internal/general.ts

```diff
@@ -130,3 +130,19 @@ export async function queryIndexer<T>(args: {
   });
   return data;
 }
+
+export async function getIndexerLastSuccessVersion(args: { aptosConfig: AptosConfig }): Promise<number> {
```

Comment:
looks like the query accepts `where` condition but we dont support it in the api/internal functions

Reply:
Will remove.

I was thinking about having it there to allow flexibility in the future, but will remove where clause for now